src/components/RequestStatus.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+type RequestStatusProps = {
+  requestStatus: RequestStatusOptions
+  message?: string | null
+  validationErrors?: string[] | null
+}
+
+const RequestStatus = ({
```

Comment:
This looks awesome! Great job on implementation. However if I really want to nitpick I'll say that it might be marginally improved with a more general name because this potentially shows validation messages as well. Maybe a name like `FormStatus` or `FormMessages`, etc?